script/lighthouse-top-pages.bash

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+project_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." >/dev/null 2>&1 && pwd )"
+
+lighthouse_paths=(
+  /books/anatomy-and-physiology/pages/1-introduction
+  /books/biology-2e/pages/1-introduction
+  /books/introduction-sociology-3e/pages/1-introduction
+  /books/psychology-2e/pages/1-introduction
+  /books/us-history/pages/1-introduction
+)
+
+lighthouse_base_report_path=/tmp/lighthouse-reports
+
+mkdir -p "$lighthouse_base_report_path"
+
+bucket_name=unified-web-lighthouse
+
+# Note: this probably fails due to pagination if there are > 1,000 prefixes
+last_prefix=$(aws s3api list-objects-v2 --bucket "$bucket_name" --delimiter / \
```

Comment:
could there be a "last-prefix.txt" file that you write to when you do the upload so that you don't have to scan the whole bucket?

Reply:
Done